_inc/client/my-plan/checklist-progress-card/progress-bar/screen-reader-text/style.scss

```diff
@@ -0,0 +1,26 @@
+// Text meant only for screen readers
+.screen-reader-text {
```

Comment:
Core has this style already for wp-admin — perhaps we don't need it at all?

https://github.com/WordPress/WordPress/blob/80aee4ae87343ea3990314c453793d334beb8ebb/wp-admin/css/common.css#L120-L135
https://github.com/WordPress/WordPress/blob/80aee4ae87343ea3990314c453793d334beb8ebb/wp-admin/css/wp-admin.css#L1

Reply:
Dropped the styles.